packages/components/pie-lottie-player/src/index.ts

```diff
@@ -0,0 +1,215 @@
+import {
+    LitElement, html, isServer, unsafeCSS,
+} from 'lit';
+import { property, query } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { LottiePlayer, AnimationItem } from 'lottie-web';
+
+import { defineCustomElement } from '@justeattakeaway/pie-webc-core';
+import { LottiePlayerProps, defaultProps } from './defs';
+
+// Valid values available to consumers
+export * from './defs';
+
+const componentSelector = 'pie-lottie-player';
+
+/**
+ * @tagname pie-lottie-player
+ */
+export class PieLottiePlayer extends LitElement implements LottiePlayerProps {
+    @query('div')
+    private _hostElement!: HTMLDivElement;
+
+    private _lottie?:LottiePlayer;
+    private _animationInstance?: AnimationItem;
+
+    private _animationSrc = '';
+    private _animationData?:object;
+    private _autoPlayDisabled = defaultProps.autoPlayDisabled;
+    private _loopDisabled = defaultProps.loopDisabled;
+    private _speed = defaultProps.speed;
+    private _direction = defaultProps.direction;
+
+    protected async firstUpdated (): Promise<void> {
+        // Check if the code is running in a browser environment
+        if (isServer) return;
```

Comment:
In theory this should be unnecessary as `firstUpdated` is only called on the client: https://lit.dev/docs/components/lifecycle/#firstupdated